inc/saiswitch.h

```diff
@@ -499,6 +499,8 @@ typedef enum _sai_switch_tunnel_attr_t
 /**
  * @brief Attribute Id in sai_set_switch_attribute() and
  * sai_get_switch_attribute() calls
+ *
+ * @flags skip enum value check
```

Comment:
this is the tag that is used to skip the enum check as the enum value must be increased >=0 and <=1

Reply:
why this is added? on attributes each attribute should increase +1